internal/repos/exclude.go

```diff
@@ -8,14 +8,14 @@ import (
 
 // excludeFunc takes a string and returns true if it should be excluded. In
 // the case of repo sourcing it will take a repository name or ID as input.
-type excludeFunc func(string) bool
+type excludeFunc func(input any) bool
 
 // excludeBuilder builds an excludeFunc.
 type excludeBuilder struct {
 	exact    map[string]struct{}
 	patterns []*regexp.Regexp
-
-	err error
+	generic  []func(repo interface{}) bool
```

Comment:
Did you mean to use the type defined above?

```suggestion
	generic  []excludeFunc
```